src/Orders.sol

```diff
@@ -5,103 +5,110 @@ import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
 
 /// @notice Contract capable of processing fulfillment of intent-based Orders.
 abstract contract OrderDestination {
-    /// @notice Emitted when an swap order is fulfilled by the Builder.
-    /// @param originChainId - The chainId on which the swap order was submitted.
+    /// @notice Emitted when an Order output is fulfilled.
```

Comment:
this is emitted for each output of the order, not the order itself?

Reply:
yeah good call. https://github.com/init4tech/zenith/pull/49/commits/b94e158c2554ad808ddea6c5a61ac2f5cb6eb5c0